src/backend/common/helpers/matchstats_helper.py

```diff
@@ -39,26 +156,28 @@ def build_team_mapping(
         team_id_map: A dict of key: team_str, value: row index in x_matrix that corresponds to the team
         """
         # Build team list
-        team_set = set()
+        team_list = set()
```

Comment:
nit: Why the rename here? This is a set - the name `team_set` feels fine, right?

Reply:
No idea how that happened, honestly.